### PR TITLE
Rework README

### PR DESCRIPTION
**Pull request by szepeviktor**
@markkovari I hope you will like it 🤩

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,22 @@
...
-  <tr>
-    <td><img src="https://github-readme-stats.vercel.app/api?username=markkovari&show_icons=true&count_private=true&theme=transparent"></td>
-    <td><img  src="https://github-readme-stats.vercel.app/api/top-langs/?username=markkovari&theme=gruvbox&count_private=true)"></td> 
-  </tr>
-</table>
-
-# A boat me
+| General | Language specific |
+| ------- | ----------------- |
+| ![GitHub Stats](https://github-readme-stats.vercel.app/api?username=markkovari&show_icons=true&count_private=true&theme=transparent) | ![Top Languages](https://github-readme-stats.vercel.app/api/top-langs/?username=markkovari&theme=gruvbox&count_private=true) |
```
Comment (szepeviktor) [inline README.md:5]: Markdown only, alt attributes.

File: README.md [hunk 2]
```diff
@@ -1,28 +1,22 @@
...
+
+## A boat me
```
Comment (szepeviktor) [inline README.md:7]: H1 > H2

File: README.md [hunk 3]
```diff
@@ -1,28 +1,22 @@
 # Stats
 
-<table>
-  <tr>
-    <th>General</th>
-    <th>Language specific</th> 
-  </tr>
-  <tr>
-    <td><img src="https://github-readme-stats.vercel.app/api?username=markkovari&show_icons=true&count_private=true&theme=transparent"></td>
-    <td><img  src="https://github-readme-stats.vercel.app/api/top-langs/?username=markkovari&theme=gruvbox&count_private=true)"></td>
```
Comment (szepeviktor) [inline README.md:10]: A stray `)` !

File: README.md [hunk 4]
```diff
@@ -1,28 +1,22 @@
...
-# A boat me
+| General | Language specific |
+| ------- | ----------------- |
+| ![GitHub Stats](https://github-readme-stats.vercel.app/api?username=markkovari&show_icons=true&count_private=true&theme=transparent) | ![Top Languages](https://github-readme-stats.vercel.app/api/top-langs/?username=markkovari&theme=gruvbox&count_private=true) |
+
+## A boat me
 
 Hi 👋 there 👋
 
-You know olny these wars are acceptable...
```
Comment (szepeviktor) [inline README.md:18]: A typo.

File: README.md [hunk 5]
```diff
@@ -1,28 +1,22 @@
...
+You know only these wars are acceptable...
+
+[![Codewars badge](https://www.codewars.com/users/markkovari/badges/small)](https://www.codewars.com/users/markkovari)
```
Comment (szepeviktor) [inline README.md:13]: Add link to the badge.

File: README.md [hunk 6]
```diff
@@ -1,28 +1,22 @@
...
+<!--
 
-In case of emergency please concact me on LinkedIn or via email. 
+In case of emergency please concact me on LinkedIn or via email.
 Chechkout my [homepage](https://markkovari-io.vercel.app/), which is heavily WIP
 
 Thank you 🙇
 
-
+-->
```
Comment (szepeviktor) [inline README.md:22]: Close HTML comment.

**Comment by markkovari** [comment]
@szepeviktor What the hack? :D Really appreciate it! Unfortunately there is only one of us who spent enough time to make things pretty and valid!

**Pull request merged**